readme.md

```diff
@@ -54,7 +66,9 @@ console.log(results);
 
 Your Answer:
 ```text
+db.authors.find({name: "Bob"})
 
+To query name of the author from a NoSQL database, you would simple run the above command. Authors being the author model.  
```

Comment:
The question was looking for *mongoose* JS code, not mongoDB code. See the solution branch for the correct answer.